src/Accounts/Authentication.ResourceManager/AzureRmProfile.cs

```diff
@@ -398,8 +398,12 @@ public bool TryGetContextName(IAzureContext context, out string name)
             name = null;
             if (context != null)
             {
-
-                if (context.Subscription != null && context.Account != null)
+                if (null != context.Tenant && context.Subscription != null && null != context.Account)
+                {
+                    name = string.Format("{0} ({1}) - {2} - {3}", context.Subscription.Name, context.Subscription.Id, context.Tenant.Id, context.Account.Id);
```

Comment:
use $""

Reply:
I would like to keep consistent with the subsequent codes